src/toil/fileStore.py

```diff
@@ -721,6 +721,13 @@ def readGlobalFile(self, fileStoreID, userPath=None, cache=True, mutable=None):
                                                               0.0, False)
         return localFilePath
 
+    def exportFile(self, jobStoreFileID, dstUrl):
+        while jobStoreFileID in self._pendingFileWrites:
```

Comment:
It's basically the same thing..... waitOnDownload waits for the harbinger to get deleted, signifying the file has been transferred to/from the jobstore. Can possibly  rename it to `waitOnTransfer`